packages/react-toolbar/src/components/Toolbar/useToolbarStyles.ts

```diff
@@ -1,7 +1,15 @@
+import { SlotClassNames } from '@fluentui/react-utilities';
 import { makeStyles, mergeClasses } from '@griffel/react';
-import type { ToolbarState } from './Toolbar.types';
+import type { ToolbarSlots, ToolbarState } from './Toolbar.types';
 
+/**
+ * @deprecated
+ */
 export const toolbarClassName = 'fui-Toolbar';
```

Comment:
For `Toolbar` you can actually remove this class name instead of deprecating it since the component is not being exported from `@fluentui/react-components` yet:

```suggestion
```